programs/cleos/main.cpp

```diff
@@ -2565,9 +2565,12 @@ int main( int argc, char** argv ) {
    getCode->add_option("-a,--abi",abiFilename, localized("The name of the file to save the contract .abi to") );
    getCode->add_flag("--wasm", code_as_wasm, localized("Save contract as wasm"));
    getCode->set_callback([&] {
+      uint64_t code_seq, abi_seq = 0;
```

Comment:
code_seq is uninitialized.

Reply:
Thank you for spotting this. I've fix it in the follow up commit.